service/service.go

```diff
@@ -242,12 +226,22 @@ func (t *service) AddThread(
 	}); err != nil {
 		return
 	}
+	if args.ReadKey != nil {
```

Comment:
As mentioned in slack, we can add a log here (like `CreateThread`) if there's a read key. Not a perfect heuristic, but works.